teps/0092-scheduling-timeout.md

```diff
@@ -160,7 +173,137 @@ This would mean at most 2x the number of timeouts for each individual `TaskRun`.
 
 This feature would be introduced [at the alpha stability level](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features).
 
-By default, when not specified, this timeout would have a value of 0, indicating that there is no scheduling timeout.
+
+Total Timeout Default:
+  - If either `Execution` or `Scheduling` timeout is provided, the `Total` timeout will default to `60` mins
+  - Otherwise, the default `Total` timeout will be calculated from `Scheduling` + `Execution`
+
+### Examples
+
+Use case: resource-constrained and wants to cancel if not scheduled quickly.
+User creates this:
+
+```yaml
+spec:
+  timeouts:
+    scheduling: "5m"
+```
+
+The spec that is calculated is this:
+
+```yaml
+spec:
+  timeouts:
+    scheduling: "5m"  # User-specified value
+    execution: "55m"  # Calculated from default total timeout - scheduling timeout
+    total: "60m"  # Default value
+```
+
+Use case: user does not care when the pod is scheduled but wants the `TaskRun` executed within a specific timeframe.
+User creates this:
+
+```yaml
+spec:
+  timeouts:
+    execution: "20m"
+```
+
+The spec that is calculated is this:
+
+```yaml
+spec:
+  timeouts:
+    scheduling: "40m"  # Calculated from default total timeout - execution timeout
+    execution: "20m"  # User-specified value
+    total: "60m"  # Default value
+```
+
+Use case: only cares about the overall timeout.
+User creates this:
+
+```yaml
+spec:
+  timeouts:
+    total: "45m"
+```
+
+The spec that is calculated is this:
+
+```yaml
+spec:
+  timeouts:
+    scheduling: "0m"  # Defaults to 0
+    execution: "45m"  # Calculated by total - scheduling
+    total: "45m"  # User-specified value
+```
+
+### Validation
+
+These are valid yaml files:
+
+```yaml
+spec:
+  timeouts:
+    execution: "55m"
+    total: "0m"
+```
+Valid: In this scenario, the pod will never timeout before the pod is scheduled, but will timeout if the runtime is not completed within 55 minutes.
+
+
+```yaml
+spec:
+  timeouts:
+    scheduling: "5m"
+    total: "0m"
+```
+Valid: In this scenario, the pod will only timeout if the pod isn't scheduled within 5 minutes.
+
+
+```yaml
+spec:
+  timeouts:
+    scheduling: "10m"
+    execution: "75m"
+    total: "0m"
+```
+Valid: In this scenario, the pod will timeout if not scheduled in 10m or runtime exceeds 75 minutes.
+
+
+```yaml
+spec:
+  timeouts:
+```
+Valid: It's valid to not provide any timeouts. In this scenario, the total timeout will default to 60 mins.
```

Comment:
It might be clearer to put this under "examples", and say "if the user doesn't specify any timeouts, here's what would be used." I don't think this is valid yaml.